doc/sailfish-webview-webview.qdoc

```diff
@@ -66,6 +68,505 @@
+  \brief The margin taken by the keyboard at the bottom of the page
+
+  When the keybaord is opened it may overlap web content displayed in a
+  \l{WebView}. Setting this value allows the web content to be arranged to
+  compensate for the overlapped area.
+
+  When using a \l{WebView} this property will be set automatically when the
+  keyboard is opened or closed.
+*/
+
+/*!
+  \qmlsignal WebView::contentOrientationChanged(Qt::ScreenOrientation orientation)
+  \brief This signal is emitted when the screen orientation of the rendered
+         content changes.
+*/
+
+/*!
+  \internal
+
+  \qmlsignal WebView::openUrlInNewWindow()
+  \brief This signal is emitted to indicate a request to open a new window.
+*/
+
+// ------------------------------------------------------ from QuickMozView.h:
+
+/*!
+  \qmlproperty bool WebView::privateMode
+  \brief Whether the webview is in private browsing mode
+
+  If private mode is enabled, history will not be stored in the device.
+
+  Note: the value for this property must be set when the webview is
+  initialized.  Any attempt to set the value of this property after the
+  webview has been instantiated will have no effect.
+*/
+
+/*!
+  \qmlproperty bool WebView::active
+  \brief Whether the webview is actively evaluating or rendering content
+
+  If a webview is not set as active, it will not display any content.
+*/
+
+/*!
+  \qmlproperty bool WebView::loaded
+  \brief Whether the webview content has finished loading successfully
+*/
+
+/*!
+  \qmlproperty Qt::ScreenOrientation WebView::orientation
+  \brief The orientation of the webview
+
+  Ordinarily, the value of this property will follow the orientation of the
+  (Silica) Page in which the webview is shown.
+*/
+
+/*!
+  \qmlproperty qreal WebView::viewportWidth
+  \brief The width of the webview viewport
+
+  Note that this will usually be the same as the width of the webview, but
+  in some cases clients may wish to set the viewport width explicitly
+  (e.g. to show some chrome at one edge or the other, without obscuring some
+  content underneath, while keeping the geometry of the webview item itself
+  unchanged).
+
+  \sa contentRect
+*/
+
+/*!
+  \qmlproperty qreal WebView::viewportHeight
+  \brief The height of the webview viewport
+
+  Note that this will usually be the same as the height of the webview, but
+  in some cases clients may wish to set the viewport height explicitly
+  (e.g. to show some chrome at one edge or the other, without obscuring some
+  content underneath, while keeping the geometry of the webview item itself
+  unchanged).
+
+  \sa contentRect
+*/
+
+/*!
+  \internal
+  \qmlproperty QObject* WebView::child
+  \brief The webview object
+
+  // TODO: why does this property exist?  Can we remove it?
+  // It was added in b900d08cf8a98c4837562b7b63c78b6ac0600548
+  // But seems mostly useless, as it just returns "this"...
+
+  // If it is required, should we rename this property to _child ?
```

Comment:
This should be removed. Let's do not document it at all. I'll create a new task for cleaning it up.

Reply:
Created JB#54340 / JOLLA-156

Reply:
removed